src/workspace/qml/MuseScore/Workspace/NewWorkspaceDialog.qml

```diff
@@ -93,6 +93,14 @@ StyledDialogView {
             }
         }
 
+        StyledTextLabel {
+            Layout.topMargin: 12
+            Layout.fillWidth: true
+
+            horizontalAlignment: Qt.AlignLeft
+            text: workspaceModel.errorMessage
```

Comment:
Shouldn't we say explicitly `visible: !isEmpty`? Or did you omit that on purpose?

Reply:
Usually, this is not a good idea, because the content of the dialog will bounce when this label appears